ads/google/a4a/utils.js

```diff
@@ -523,3 +489,53 @@ export function mergeExperimentIds(newIds, currentIdString) {
   return currentIdString + (currentIdString && newIdString ? ',' : '')
       + newIdString;
 }
+
+/**
+ * Adds two CSI signals to the given amp-analytics configuration object, one
+ * for render-start, and one for ini-load.
+ *
+ * @param {!Window} win
+ * @param {!Element} element The ad slot.
+ * @param {!JSONType} config The original config object.
```

Comment:
I think JSONType is what AmpAnalytics uses as a type for its configuration object. https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/amp-analytics.js#L67